### ISSUE TITLE
Attribute names should not have to be unique 

### ISSUE THREAD

**Issue by mnaugle-tt** [tt-acm/Spectacles.GrasshopperExporter#23]
@anagpuyol 

We should allow an attribute name to be repeated, this would be my hacky way of getting around issue #22 (which I shouldn't have to do, but we should allow what is shown in the screen shot if someone wanted to do it this way for some reason).

![image](https://cloud.githubusercontent.com/assets/3228216/11305899/012fe8ac-8f7d-11e5-999a-37f311b81f67.png)

**Comment by anagpuyol** [comment]
@bhowes-tt do you think that adding more than one value to the same attribute name could be an issue on the web side? I'm thinking in terms of retrieving values by key (attribute name)...?

**Comment by bhowes-tt** [comment]
@anagpuyol - I'm not 100% sure but I think it would be fine on the web side.

In general I think this is weird...  @mnaugle-tt --- why do we need to allow multiple attributes of the same name?  Is the component just less error prone that way?  

I can't imagine a useful scenario, but if you just want the components to solve, I might be able to stomach it.  But, let the record show that it feels like pushing :hankey: downhill to me ;]

**Comment by mnaugle-tt** [comment]
yes this was mainly about having it solve more easily for those that don't understand grasshopper iteration (and especially datatree's if that is the way to handle this per #22 )
